devs/docs/es-backports.rst

```diff
@@ -353,8 +353,8 @@ should be crossed out as well.
 - [ ] c2deb287f13 Add a cluster setting to disallow loading fielddata on _id field (#49166)
 - [ ] 563b2736a9f Annotated text type should extend TextFieldType (#49555)
 - [ ] 7069045cd42 Add the simple strategy to cluster settings (#49414)
-- [ ] 725dda37ea5 Flush instead of synced-flush inactive shards (#49126)
-- [ ] b8ce07b4cc5 Pre-sort shards based on the max/min value of the primary sort field (#49092)
+- [d] 725dda37ea5 Flush instead of synced-flush inactive shards (#49126) -- CrateDB 5.0
```

Comment:
shouldn't this be `[x]` (applied)?

Reply:
It's a bit confusing in this case - see the commit https://github.com/crate/crate/pull/10758/commits/364cce50dc396ccdac6f3a259644464aa52369b3

I applied the ES 7.6 variant but we'll still have to apply the mentioned commit 725dda37ea5 - which is the ES 8.0 variant.

Reply:
ah got it, thx.